src/library/parserm3u.cpp

```diff
@@ -87,7 +80,7 @@ QList<QString> ParserM3u::parse(const QString& filename) {
     }
 
     QFileInfo fileInfo(filename);
-    const QStringList fileLines = fileContents.split('\n');
+    const QStringList fileLines = fileContents.split(QRegExp("\r\n|\r|\n"));
```

Comment:
Deprecated in Qt6, the docs recommend to use QRegularExpression instead: https://doc.qt.io/qt-6/qregexp.html#details

Reply:
Should also be extracted into a constant and compiled only once if the matching is repeated on different inputs.

Reply:
Done.